WordPress/Classes/ViewRelated/Plugins/PluginDirectoryViewModel.swift

```diff
@@ -36,19 +36,11 @@ class PluginDirectoryViewModel: Observable {
         newReceipt = store.query(.feed(type: .newest))
 
         actionReceipt = ActionDispatcher.global.subscribe { [changeDispatcher, throttle] action in
-            switch action {
-            case PluginAction.receivePlugins,
-                 PluginAction.receiveFeaturedPlugins,
-                 PluginAction.receivePluginDirectoryFeed,
-                 PluginAction.receivePluginDirectoryEntry:
-                // Fairly often, a bunch of those network calls can finish very close to each other — within few hundred
-                // milliseconds or so. Doing a reload in this case is both wasteful and noticably slow.
-                // Instead, we throttle the call so we trigger the reload at most once a second.
-                throttle.throttle {
-                    changeDispatcher.dispatch()
-                }
-            default:
-                break
+            // Fairly often, a bunch of those network calls can finish very close to each other — within few hundred
```

Comment:
Listening to those four actions is not enough — it was dropping updates to local plugins, for example. Keeping this in-sync with what happens internally in a store would be a maintenance nightmare, and the `throttle` solves the issue anyway, so i got rid of it.

Reply:
Good switch. I initially subscribed to actions so I could choose which one I want to know about, and didn't think to switch back to to store changes when I realised that didn't work.